zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -305,11 +306,12 @@ public ZeppelinServer() throws Exception {
     this.schedulerFactory = new SchedulerFactory();
 
     this.replFactory = new InterpreterFactory(conf, notebookServer);
-    Class<?> notebookStorageClass = getClass().forName(
+    /*Class<?> notebookStorageClass = getClass().forName(
```

Comment:
It's much better to delete code that is not used instead of commenting it - that is what version control system is for.